src/tests/system/tests/test_sudo.py

```diff
@@ -556,3 +556,35 @@ def test_sudo__local_users_negative_cache(client: Client, provider: LDAP, sssd_s
 
     result = client.tools.tshark(["-r", "/tmp/sssd.pcap", "-V", "-2", "-R", "ldap.filter"])
     assert "uid=user-1" not in result.stdout
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.authorization
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_sudo__defaults_rule(client: Client, provider: GenericProvider):
```

Comment:
I don't think the name properly describes what this is testing.  How about 

test_sudo__user_allowed_without_a_password 
or
test_sudo__user_allowed_with_nopasswd_all_set


Since the sudo config is 
```
%wheel	ALL=(ALL)	NOPASSWD: ALL
```

Reply:
Here the test case is testing for effects of 'defaults' entry in SUDOers container in directory-server.  As per the man page, it's(cn=defaults) an entry in SUDOers container. Not a rule in strict sense. It's a specific scenario. Check the PR description or man 5 sudoers.ldap for context.

> I don't think the name properly describes what this is testing. How about
> 
> test_sudo__user_allowed_without_a_password or test_sudo__user_allowed_with_nopasswd_all_set
No, these name do not even hint toward if  the effect of 'defaults' entry/rule on other is being tested. The test case name could be 'test_sudo__defaults_non_authentication_for_rule_with_authentication'
> 
> Since the sudo config is
> 
> ```
> %wheel	ALL=(ALL)	NOPASSWD: ALL
> ```
This above rule is defined locally and will be applicable to members of 'wheel' group. It's irrelevant in current PR context.
'cn=defaults' can be explored with different inputs/options.

Reply:
'test_sudo__defaults_rule_with_nopasswd' ?

Reply:
'test_sudo__defaults_non_authentication_for_rule_with_authentication' is still a little confusing, but it's more informative. Now that I understand the test better, how about, 'test_sudo__change_sudorole_default_cn_removing_sudo_command_authentication' ?

Reply:
Added new sudorule name